libraries/chain/controller.cpp

```diff
@@ -1667,7 +1667,7 @@ struct controller_impl {
 
       emit( self.block_start, head->block_num + 1 );
 
-      if (auto dm_logger = get_deep_mind_logger()) {
+      if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
At `start_block`, we don't have transaction type information.

Reply:
Yep, I got a bit copy/paste happy on my comments.